src/components/common/CaseDetails/CaseDetails.js

```diff
@@ -28,6 +28,11 @@ function CaseDetails() {
   const [filterSearch, setFilterSearch] = useState('');
   const [filteredResults, setFilteredResults] = useState([]);
 
+  //State slice for notes
+  const [notes, setNotes] = useState([
```

Comment:
Can we import in the notes from the backend using GET `/:clientId/notes` next?

Reply:
For sure!